x-pack/plugins/security_solution/public/timelines/components/timeline/session_tab_content/use_session_view.tsx

```diff
@@ -224,6 +225,8 @@ export const useSessionView = ({
   const { globalFullScreen } = useGlobalFullScreen();
   const { timelineFullScreen } = useTimelineFullScreen();
 
+  const { canAccessEndpointManagement } = useUserPrivileges().endpointPrivileges;
```

Comment:
Didn't know about this one, nice!

Reply:
What does using the `canAccessEndpointManagement` accomplish?

Reply:
The ansi message will contain a link to /security/administration/policy which is a listing of the defend integration policies. Only users with this permission can access it. I'm passing it as a parameter since I can't import this hook into session_view as it would introduce a circular dependency.